tutorials/streamlit_notebooks/healthcare/DEID_PHI_TEXT.ipynb

```diff
@@ -254,17 +269,17 @@
     },
     {
       "cell_type": "code",
+      "execution_count": 5,
       "metadata": {
         "id": "cK9xxkkfrsLc"
       },
+      "outputs": [],
       "source": [
         "# Change this to the model you want to use and re-run the cells below.\n",
         "# Anatomy models: ner_anatomy\n",
         "MODEL_NAME = \"ner_deid_large\"\n",
         "DEID_MODEL_NAME = \"deidentify_large\""
-      ],
```

Comment:
We should use subentity_i2b2 and generic_augmented models here. And please visualize all results on pandas df by setting display.

Reply:
ner_deid_subentity_augmented_i2b2 is ok

DeIdentificationModel.pretrained() doesnt work with **ner_deid_generic_augmented** 
because this is not MedicalNerModel so I couldn't change